tests/integration/fixtures.py

```diff
@@ -797,6 +798,23 @@ def _create_function():
             LOG.debug(f"Unable to delete policy {policy_arn=} in cleanup")
 
 
+@pytest.fixture
+def check_lambda_logs(logs_client):
+    def _check_logs(func_name: str, expected_lines: List[str] = None):
```

Comment:
why did you decide to nest the function definitions like this? any reason not to have a single function that takes all the args?

Reply:
Actually just moved this fixture from `test_lambda.py` to `fixtures.py` so that I could reuse it in `test_lambda_integration`